### ISSUE TITLE
Keyboard shortcut to close sidebar

### ISSUE THREAD

**Issue by dwhly** [hypothesis/product-backlog#1116]
### Problem you are trying to address with this feature
There's currently no direct keyboard shortcut to close the sidebar.  You *can* tab to the chevron in the UX and enter to activate it, which will close the sidebar if opened, but it would be helpful to have a specific shortcut to do this.

### Your solution
Map [ESC] to closing the sidebar.

**Comment by maciejmotyka** [comment]
Yes, please. I use Vimium to select the text and annotate it - all done via keyboard. Closing the sidebar and giving focus back to the page is the only activity that needs a mouse intervention.

**Comment by jorgebr** [comment]
Yes, please!  Would be so good.

**Comment by vasudev-sharma** [comment]
Likewise, that feature would be really great as it gets distracting to only use mouse for closing sidebar

**Comment by apropername** [comment]
i agree
